packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -521,3 +525,29 @@ gmaps.LatLng _pixelToLatLng(gmaps.GMap map, int x, int y) {
 
   return projection.fromPointToLatLng!(point)!;
 }
+
+String _sanitizeHtml(String htmlString) {
+  TrustedTypePolicy? trustedTypePolicy;
+
+  try {
+    // Firefox and Safari don't support Trusted Types yet.
+    // See https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypePolicyFactory#browser_compatibility
+    trustedTypePolicy =
+        WindowWithTrustedTypes(window).trustedTypesNullable?.createPolicy(
```

Comment:
This is re-creating the `trustedTypesPolicy` for every call to `_sanitizeHtml`, but there's no need to do that, you can just create this policy the first time it's needed, and then reuse it on every call to `_sanitizeHtml`.

Reply:
Done